### DISCUSSION TITLE
How to set _config_mode

### DISCUSSION THREAD

**Discussion by prajgujarathi** [google/digitalbuildings#586] [Q&A] [answered]
I keep getting errors on valid ymal files saying KeyError: 'No valid _config_mode is set' when trying to validate using instance_validator

**Comment by charbull** [comment]
Hi @prajgujarathi,

we are looking into it, can you please create an issue with some data for us to reproduce?

@trav3711 @mschulze17 can we please create a task to check.

Thank you

**Comment by mschulze17** [comment, marked as answer]
See #594 for resolution

**Reply by prajgujarathi** [reply]
I do not see any resolution on that issue

**Reply by mschulze17** [reply]
Apologies - this comment is just meant to document that the topic of this discussion has been migrated to an open Issue
